src/components/LoginFields.js

```diff
@@ -14,8 +17,11 @@ const LoginFields = () => {
       name,
     }).unwrap()
       .then((response) => {
+        console.log(response);
         if (response.message === 'Logged In Successfully') {
-          navigate('/home', { replace: true });
+          dispatch(updateUser({ loggedIn: true, data: response.user }));
+          window.location.reload();
+          navigate('/', { replace: true });
```

Comment:
Is there a need of reloading the page (`window.location.reload()`) if you're going to redirect the user to the root route (`navigate('/', { replace: true }`)?